pkg/provenance/provenance.go

```diff
@@ -139,26 +157,67 @@ func NewSpec() *Spec {
 
 func FindProvenanceObjectByName(name string, allObjects []ProvenanceOfObject) *ProvenanceOfObject {
 	for _, value := range allObjects {
+		print(value.Name)
```

Comment:
Is this stray print that was added for debugging purpose?

If so, it can be removed.

Reply:
Ok